apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SoftwareCheckStep.tsx

```diff
@@ -41,13 +44,13 @@ const SoftwareCheckStep = ({ isDisplayed, onComplete, productName }: Props) => {
   const device = useSelector(getCurrentDevice);
   const deviceId = device?.deviceId ?? "";
 
-  const { genuineState, devicePermissionState, error, resetGenuineCheckState } = useGenuineCheck({
+  const { genuineState, devicePermissionState, resetGenuineCheckState } = useGenuineCheck({
     getGenuineCheckFromDeviceId: getGenuineCheckFromDeviceIdCommand,
     isHookEnabled: genuineCheckStatus === SoftwareCheckStatus.active,
     deviceId,
   });
 
-  const { latestFirmware /*, error */, status } = useGetLatestAvailableFirmware({
+  const { latestFirmware /*, error */, status, lockedDevice } = useGetLatestAvailableFirmware({
```

Comment:
I know it's not your doing but, why do we keep this error there if not used?

Reply:
true, thanks ! going to clean that